oml/registry/losses.py

```diff
@@ -22,8 +25,20 @@ def get_criterion(name: str, **kwargs: Dict[str, Any]) -> nn.Module:
         return LOSSES_REGISTRY[name](**kwargs)
 
 
-def get_criterion_by_cfg(cfg: TCfg) -> nn.Module:
+def get_criterion_by_cfg(
```

Comment:
We added possibility to use `runtime_kwargs` in samplers registry. Is it possible to do it in the same way?